broker-cli/commands/health-check.js

```diff
@@ -26,6 +26,92 @@ const ENGINE_STATUS_CODES = Object.freeze({
   VALIDATED: 'VALIDATED'
 })
 
+/**
+ * Prints health check status summary in Table format.
+ * @param {Object} engineStatus
+ * @param {Object} orderbookStatus
+ * @param {Object} relayerStatus
+
+ * @returns {Void}
+ */
+
+function createHealthCheckTable (engineStatus, orderbookStatus, relayerStatus) {
```

Comment:
is there any way we can refactor this and the json function? it seems like a lot of duplicated code